config/locales/en/shared.yml

```diff
@@ -319,10 +299,39 @@ en:
           shared_ownership_with_partner: The home is owned with someone else?
           percentage_home: Percentage of the home legally owned by the client
           percentage_home_with_partner: Percentage of the home legally owned by the client and their partner
+      client_bank_accounts:
+        card_heading: Your client's accounts
+        aria_label: Your client's accounts
+        current_accounts: Current accounts
```

Comment:
Some of these are plural ("Current accounts", "Savings accounts") and some are singular ("Current account", "Savings account"). Is this intentional?

Reply:
This was inconsistent previously but yes I can see it is still. Have asked quay for definitive answer on what it should be and will apply

![Screenshot 2025-03-10 at 12 37 47](https://github.com/user-attachments/assets/de1b7595-f8fd-4493-9d51-197805c8a038)

Reply:
designer confirms it should be 
- Current accounts
- Savings accounts